website/content/partials/known-issues/config_listener_proxy_protocol_behavior_issue.mdx

```diff
@@ -0,0 +1,33 @@
+<a id="listener-proxy-protocol-config" />
+
+### Potential DoS when using the `deny_unauthorized` proxy protocol behavior for a TCP listener
+
+#### Affected versions
+
+##### Community Edition (CE)
+
+- 1.10.x - 1.15.x
+- 1.17.1
+
+##### Enterprise
+
+- 1.10.x+ent - 1.15.11+ent
+- 1.16.5+ent
+- 1.17.1+ent
+
+#### Issue
+
+When a Vault TCP listener is configured with the `deny_unauthorized` [proxy_protocol_behavior](/vault/docs/configuration/listener/tcp#proxy_protocol_behavior)
+and receives a request from an untrusted upstream connection, Vault will no longer
+respond to requests via that listener.
+
+This could lead to a situation where the Vault API could be vulnerable to a denial-of-service
+(DoS) attack, if requests are intentionally made by untrusted upstream connections
+to the Vault API, causing it to become unavailable on that node.
+
+This issue is addressed in Vault 1.15.12+ent, 1.16.6+ent, 1.17.2, 1.17.2+ent and later.
```

Comment:
```suggestion
The fact that the listener takes the time to process the request before ignoring it, leaves the listener vulnerable to denial-of-service (DoS) attacks by requests intentionally sent from untrusted upstream connections.

We recommend upgrading to Vault 1.15.12+ent, 1.16.6+ent, 1.17.2, 1.17.2+ent or later to address the vulnerability.
```

Style correction: write in active voice

Reply:
> The fact that the listener takes the time to process the request before ignoring it, leaves the listener 

The vulnerability is such that you can prevent Vault from serving its API entirely - therefore no further requests can be successful to that Vault node. 

It's not that the listener takes a long time to process anything, it doesn't it, it's quick to process but closes when we stop serving HTTP in response to the way the library returns the error from the listener's `Accept` method.

Reply:
> We recommend upgrading to Vault 1.15.12+ent, 1.16.6+ent, 1.17.2, 1.17.2+ent or later to address the vulnerability.

These versions aren't out yet, so I was just following the pattern we've used in previous known issues to call out the versions that (will) fix it.